test/testpy/__init__.py

```diff
@@ -53,11 +53,15 @@ def AfterRun(self, result):
     # delete the whole tmp dir
     try:
       rmtree(self.tmpdir)
+      if "NODE_PIPE_DIR" in os.environ:
+        rmtree(join(os.environ["NODE_PIPE_DIR"], 'NodePipeTmp'));
```

Comment:
Can we avoid the repetition of `join(os.environ["NODE_PIPE_DIR"], 'NodePipeTmp')` by having a `self.pipeTmpDir` member variable?